starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,63 @@
 // Names and Input
+let hacker1 = "Charpentier"
+console.log("The driver's name is " + hacker1);
 
+let hacker2 = prompt("What's your name?");
+console.log("The navigator's name is " + hacker2);
 
 //Conditionals
+if (hacker1.length > hacker2.length) {
+    console.log("The Driver has the longest name, it has " + hacker1.length + " characters");
+} else if (hacker1.length === hacker2.length) {
+    console.log("wow, you both got equally long names, " + hacker1.length + " characters!!");
+} else {
+    console.log("Yo, navigator got the longest name, it has " + hacker2.length + " characters");
+}
 
+// Print name
+let lettres = ""
+
+for(i = 0; i < hacker1.length; i++) {
+    console.log(lettres += hacker1.charAt(i).toUpperCase() + " ")
+}
+
+for(i = hacker2.length; i >= 0; i--) {
+    console.log(lettres += hacker2.charAt(i))
```

Comment:
Here, lettres has kept in memory the hacker1 letters because the scope is the same and you use +=. Maybe try this in to separate functions or assign a new variable, or start back letter empty :-)